src/encoded/schemas/library.json

```diff
@@ -266,6 +267,21 @@
             "type": "string",
             "linkTo": "Biosample"
         },
+        "mixed_biosamples": {
+            "title": "Mixed biosamples",
+            "description": "Two or more biosamples mixed together before isolating nucleic acid for library generation.",
```

Comment:
I am not sure the samples are mixed prior to the extraction of nucleic acid. Unless you are confident that is what they are doing in LRGASP, I think you should make more general statement, like "biosamples nucleic acid of which was used to construct the library".

Reply:
That makes sense, so I reworded it to be more ambiguous